services/rule_engine/src/main/java/agents/Moli.java

```diff
@@ -19,11 +19,16 @@
 import java.util.List;
 import com.mindsmiths.emailAdapter.NewEmail;
 import com.mindsmiths.emailAdapter.EmailAdapterAPI;
+import com.mindsmiths.gsheetsAdapter.GSheetsAdapterAPI;
+import com.mindsmiths.gsheetsAdapter.reply.Spreadsheet;
 
 @Data
 @ToString(callSuper = true)
 @NoArgsConstructor
 public class Moli extends Agent {
+
+    String email = "test@gmail.com";
```

Comment:
You don't need a variable `email`. It is stored as a connection, and you can get it with `getConnection("email")`